soc/xtensa/nxp_adsp/imx8ulp/Kconfig.defconfig.series

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NXP_IMX8ULP
+
+config SOC_SERIES
+	string
+	default "imx8ulp"
```

Comment:
same as the above - needs some clean-ul, see https://github.com/zephyrproject-rtos/zephyr/pull/62974/commits/cab0759282fc96515fa93776d20469552a82eccf

Reply:
Compared to the imx8 and imx8m, this is aligned with them in the main branch.

Reply:
@TangleZ I also agree with @LaurentiuM1234 we should clean up the config and `memory.h` file.
Please use these commits as examples:
- https://github.com/zephyrproject-rtos/zephyr/pull/62974/commits/9a293e50cd9ee6a3b8d776f197f5dc206bd61bfb
- https://github.com/zephyrproject-rtos/zephyr/pull/62974/commits/1c531389d109f69c2de4c43350f3293749c1314d

Reply:
Updated.